docs/v0.7/zh/user-guide/operations/region-migration.md

```diff
@@ -6,45 +6,53 @@ Region 迁移允许用户在 Datanode 间移动 Region 数据。
 该功能仅在 GreptimeDB 集群模式下可用，并且需要满足以下条件
 - 使用 Kafka WAL
 - 使用共享存储 (例如：AWS S3)
+
 我们无法在任何上述以外的情况下使用 Region 迁移。
 :::
 
-## 寻找 MetaSrv leader
-首先我们需要找 MetaSrv Leader 节点，Region 迁移需要在 MetaSrv Leader 上执行。
-```bash
-curl {YOUR_META_SRV_SERVER}:3002/admin/leader
-# 10.244.0.157:3002
-```
 
 ## 查询 Region 分布
 
-随后我们需要查询一下表的 Region 分布，这里我们使用上一步获取的 MetaSrv Leader 的地址(`YOUR_META_SRV_LEADER_SERVER`) 。
-```bash
-curl {YOUR_META_SRV_LEADER_SERVER}:3002/admin/route?table_id={TABLE_ID}
+首先我们需要查询该数据表分区（Region）分布情况，即查询数据表中的 Region 分别在哪一些 Datanode 节点上。
 
-# "{\"type\":\"physical\",\"region_routes\":[{\"region\":{\"id\":4668629450752,\"name\":\"\",\"partition\":{\"column_list\":[],\"value_list\":[\"\\\"MaxValue\\\"\"]},\"attrs\":{}},\"leader_peer\":{\"id\":2,\"addr\":\"greptimedb-datanode-2.greptimedb-datanode.my-greptimedb:4001\"},\"follower_peers\":[]}],\"version\":2}"
+```sql
+select b.peer_id as datanode_id,
+       a.greptime_partition_id as region_id
+from information_schema.partitions a left join information_schema.greptime_region_peers b
+on a.greptime_partition_id = b.region_idwhere a.table_name='migration_target' order by datanode_id asc;
 ```
-例如：我们有以下的 Region 分布
 
-| region id     | leader_peer                                                  | leader_peer_id |
-|---------------|--------------------------------------------------------------|----------------|
-| 4668629450752 | greptimedb-datanode-2.greptimedb-datanode.my-greptimedb:4001 | 2              |
+例如：我们有以下的 Region 分布
```

Comment:
@nicecui in docs sometimes we use "we", sometimes we use "you". Perhaps we need a guide to use consistent tone.

Reply:
Got it, we will consider this in the documentation style guide.